Modules/readline.c

```diff
@@ -1019,10 +1019,16 @@ on_hook(PyObject *func)
 static int
 #if defined(_RL_FUNCTION_TYPEDEF)
 on_startup_hook(void)
+{
+#elif defined(__APPLE__) && defined(WITH_EDITLINE)
```

Comment:
Is this an _editline_ issue, or _editline_ on macOS issue? Did you test with _editline_ on Linux?

Reply:
> editline on macOS issue?

I just detected the issue with macOS only.
If you discover the Linux distro issue too, we can fix it with a separate PR.
But I don't want to make any side-effects to other platforms with this patch.